acceptance_tests/features/steps/view_collection_exercise_details_ready_for_review_state.py

```diff
@@ -74,12 +66,6 @@ def load_collection_instruments(_):
 
 @then('the status of the collection exercise is Ready for Review')
 def ce_details_state_is_ready_for_review(_):
-    ce_state = collection_exercise_details.get_status()
-    # Status updated async so wait until updated
-    for i in range(5):
-        if collection_exercise.is_ready_for_review(ce_state):
-            break
-        time.sleep(1)
+    if not browser.is_text_present('Ready for review', wait_time=2):
```

Comment:
Not sure if this will work because the page won't be updated asynchronously although the data is